src/Playwright/Transport/ApiZone.cs

```diff
@@ -31,7 +31,7 @@ namespace Microsoft.Playwright.Transport;
 internal class ApiZone
 {
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
-    public string ApiName { get; set; }
+    public string? ApiName { get; set; }
 
-    public List<StackFrame> Frames { get; set; }
+    public List<StackFrame> Frames { get; set; } = null!;
```

Comment:
NOTE: `!null` in this context means the same as using the `required` keyword. We can't use the `required` keyword tho since its not available with our TFM we support (netstandard2.0).

Reply:
What is `TFM we support` ?

Reply:
Thats our TFM: https://github.com/microsoft/playwright-dotnet/blob/70f3aa3c6d63ba886a8334f90887062b89070766/src/Playwright/Playwright.csproj#L10

Its supported only in net7+